sceptre/config/reader.py

```diff
@@ -467,8 +483,11 @@ def _render(self, directory_path, basename, stack_group_config):
             try:
                 config = yaml.safe_load(rendered_template)
             except Exception as err:
+                debug_file_path = self._write_debug_file(rendered_template)
+
                 raise ValueError(
-                    "Error parsing {}:\n{}".format(abs_directory_path, err)
+                    f"Error parsing {abs_directory_path}:\n{err}\n"
```

Comment:
the `abs_directory_path` only provides the path to the folder containing the sceptre configs.  If there are many config files in the folder how would the user know which config and stack failed to render?  Is there a way to tell users the exact config file and stack that failed to render?

Reply:
@zaro0508 you can see that above in the screenshot I already provided. The exception already tells us which file it couldn't render; it just doesn't give us any clue about "why". After merging this feature, it will become very easy for the user to figure what went wrong. Compared to now, where it's basically impossible to know.

Reply:
Oops, now I see your point. That's actually legacy behaviour you are referring to, and not part of my change, however I've now updated it anyway to add the basename. So now you'll see this:
<img width="676" alt="Screen Shot 2023-04-22 at 9 41 40 pm" src="https://user-images.githubusercontent.com/2305330/233782353-5e31b097-5017-4cf8-ae26-e2ae450635b6.png">